zerver/views/auth.py

```diff
@@ -719,6 +750,22 @@ def login_page(
     # that disables the default behavior of redirecting logged-in users to the
     # logged-in app.
     is_preview = 'preview' in request.GET
+
+    extra_context = kwargs.pop('extra_context', {})
+    extra_context["next"] = next
+
+    if extra_context.get("action_key") is None:
+        extra_context["action_key"] = ''
```

Comment:
do we need this? Can't action_key simply be absent in the dictionary?

Reply:
I initially tried it without this but the template render is skipping it without rendering i.e without this the HTML looks like `action_key = "{{ action_key}}"`. I think Jinja2 should render it as an empty string but I'm not sure why this is being skipped. So, as a kind of workaround, I added this code.

Reply:
Will be cleaner to add a conditional in the template to just skip this thing from the form if not action_key.